skynet/modules/ttt/openai_api/app.py

```diff
@@ -40,6 +40,9 @@ def initialize():
 
 
 async def is_ready():
+    if llama_path.startswith('oci://'):
```

Comment:
we could extract this into a `use_oci` since we reuse this logic in the processor